pkg/network/ebpf/c/tracer/stats.h

```diff
@@ -23,6 +23,11 @@ static __always_inline __u64 offset_rtt_var();
 #endif
 
 static __always_inline conn_stats_ts_t *get_conn_stats(conn_tuple_t *t, struct sock *sk) {
+    conn_stats_ts_t *cst = bpf_map_lookup_elem(&conn_stats, t);
```

Comment:
this is now the same as L36 right? could we just return the empty connection stat object in that case?

Reply:
Yes, this is just trying to avoid making a `bpf_map_update_with_telemetry` call, as they often return errors and skew the telemetry for those calls.

Reply:
Actually this seems to cause a correctness issue. With this change, I see cookie values that keep changing for the same connection, leading to wildly incorrect values for some connections. After reverting this, this behavior goes away.